server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -2936,6 +2948,11 @@ public DiskOffering createDiskOffering(final CreateDiskOfferingCmd cmd) {
             throw new InvalidParameterValueException("Disksize is not allowed for a customized disk offering");
         }
 
+        // check if valid cache_mode parameter
+        if(isCacheModeInvalid(cmd.getCacheMode())){
+            throw new InvalidParameterValueException("Please specify a valid cache mode parameter");
```

Comment:
Can you please add more information on the log? Example: `String.format("Invalid cache mode (%s), specify a valid cache mode parameter", cmd.getCacheMode())`